modules/op-installing-tekton-hub-with-login-and-rating.adoc

```diff
@@ -16,19 +16,22 @@ You can install {tekton-hub} on your cluster with custom configuration that supp
 [discrete]
 .Procedure 
 
-. Create a fork of the link:https://github.com/tektoncd/hub[Tekton Hub] repository.
-
-. Clone the forked repository.
+. Configure the route or ingress for {tekton-hub}.
++
+[source,terminal]
+----
+$ oc apply -f hub/config/04-openshift/40-api-route.yaml -f hub/config/04-openshift/40-auth-route.yaml -n openshift-pipelines
+----
 
 . Create an OAuth application with your Git repository hosting provider, and note the Client ID and Client Secret. The supported providers are GitHub, GitLab, and BitBucket.
 
-** For a link:https://docs.github.com/en/developers/apps/creating-an-oauth-app[GitHub OAuth application], set the Homepage URL and the Authorization callback URL as `<auth-route>`.
+** For a link:https://docs.github.com/en/developers/apps/creating-an-oauth-app[GitHub OAuth application], set the Homepage URL and the Authorization callback URL as `<auth-route-for-tekton-hub>`.
 
-** For a link:https://docs.gitlab.com/ee/integration/oauth_provider.html#user-owned-applications[GitLab OAuth application], set the `REDIRECT_URI` as `<auth-route>/auth/gitlab/callback`.
+** For a link:https://docs.gitlab.com/ee/integration/oauth_provider.html#user-owned-applications[GitLab OAuth application], set the `REDIRECT_URI` as `<auth-route-for-tekton-hub>/auth/gitlab/callback`.
 
-** For a link:https://support.atlassian.com/bitbucket-cloud/docs/use-oauth-on-bitbucket-cloud[BitBucket OAuth application], set the `Callback URL` as `<auth-route>`.
+** For a link:https://support.atlassian.com/bitbucket-cloud/docs/use-oauth-on-bitbucket-cloud[BitBucket OAuth application], set the `Callback URL` as `<auth-route-for-tekton-hub>`.
 
-. Edit the `<tekton_hub_repository>/config/02-api/20-api-secret.yaml` file of your cloned repository to include the {tekton-hub} API secrets:
+. Edit the `<tekton_hub_root>/config/02-api/20-api-secret.yaml` file to include the {tekton-hub} API secrets:
```

Comment:
Can we provide the link to upstream hub repository i.e. https://github.com/tektoncd/hub ? to update the file as now user don't need to clone the repository anymore